_data/simulations/spheharm_1d_bridge/meta.yaml

```diff
@@ -0,0 +1,51 @@
+benchmark:
+  id: 1d
+  version: "1"
+metadata:
+  # Describe the runtime environment, hardware and software
+  summary: PF Hub benchmark 1d with spherical harmonics method    #
+  author:
+    first: Arten
+    last: Tomillo
+    email: "artem.tomilo@mailbox.tu-dresden.de"
+    github_id: bridgeArchitect
+  timestamp: "23 February, 2023"
+  implementation:
+    name: custom
+    repo:
+      url: 'https://gitlab.com/bridgearchitect/speharsol'
```

Comment:
@bridgeArchitect: In the renderings, e.g., https://gitlab.com/bridgearchitect/speharsol/-/blob/main/results/sphere1000.png, there is a thin wedge missing. I assume this is a rendering artifact and not an actual disclination in the mesh?

Reply:
Dear Jonathan Guyer,

Yes, it is correct. This artifact is caused by a feature of the library Plots.jl in the programming language Julia.

Best wishes,
Artem Tomilo